crates/matrix-sdk-ui/tests/integration/room_list.rs

```diff
@@ -508,7 +469,7 @@ async fn test_sync_resumes_from_terminated() -> Result<(), Error> {
         },
         respond with = (code 400) {
             "error": "foo",
-            "errcode": "M_UNKNOWN",
+            "errcode": "M_UNKNOWN_POS",
```

Comment:
I'm not interested by a reset here, so a `M_UNKNOWN` is perfectly fine. Please revert (same for edits below) :-).

Reply:
Alright, I thought it was a typo, but dropping the commit then :)